contracts/ProviderStrategy.sol

```diff
@@ -28,27 +43,21 @@ contract ProviderStrategy is BaseStrategy {
     bool public takeProfit;
     bool public investWant;
 
-    constructor(address _vault, address _joint) public BaseStrategy(_vault) {
-        _initializeStrat(_joint);
+    constructor(address _vault) public BaseStrategy(_vault) {
+        _initializeStrat();
     }
 
     function initialize(
         address _vault,
         address _strategist,
         address _rewards,
-        address _keeper,
-        address _joint
+        address _keeper
     ) external {
         _initialize(_vault, _strategist, _rewards, _keeper);
-        _initializeStrat(_joint);
+        _initializeStrat();
     }
 
-    function _initializeStrat(address _joint) internal {
-        require(
-            address(joint) == address(0),
-            "ProviderStrategy already initialized"
-        );
-        joint = _joint;
+    function _initializeStrat() internal {
```

Comment:
I removed it, because the joint is initialized with the provider strategies, so the provider strategies must be initialized first. I'm doing this so the joint access control modifiers can be inheritied from the provider strategies.